### PR TITLE
Run TCP integration tests with grunt-jasmine-firefoxaddon

### PR DESCRIPTION
**Pull request by agallant**
Passes locally, runs on Shippable but times out on `serves multiple clients` and `wrapper connectionsCount`. Will investigate/poke to see if it resolves, but wanted to open pull request for consideration. Thanks!

[Review on Reviewable](https://reviewable.io/reviews/uproxy/uproxy-lib/289)

### CODE REVIEW

File: .shippable.yml [hunk 1]
```diff
@@ -18,4 +18,4 @@ install:
   - "firefox -v || echo 'No firefox installed.'"
   - "shippable_retry ./setup.sh install"
 script:
-  - "grunt test"
+  - "grunt test --firefox-bin /usr/bin/firefox"
```
Comment (trevj) [inline .shippable.yml]: Was this to workaround path lookup issues on Linux?

**Comment by trevj** [comment]
Our Shippable setup has been busted for some time, unfortunately, so I'm not too worried about that. Many many thanks for sending this, it looks great to me. :+1:

**Comment by trevj** [comment]
Related issue:
https://github.com/uProxy/uproxy/issues/840

File: .shippable.yml (hunk 1 above)
Comment (soycode) [inline .shippable.yml]: Yeah exactly - jpm currently uses hardcoded default paths for Firefox that are actually incorrect on Debian/Ubuntu systems. See: https://github.com/canuckistani/fx-runner/pull/1

**Pull request merged**